infra/src/webapp.tf

```diff
@@ -64,6 +142,34 @@ locals {
 
     INTERNAL_STORAGE_CONNECTION_STRING = module.storage_account.primary_connection_string
 
+    # JIRA integration for Service review workflow
+    JIRA_NAMESPACE_URL                  = var.jira_namespace_url
+    JIRA_PROJECT_NAME                   = var.jira_project_name
+    JIRA_TOKEN                          = data.azurerm_key_vault_secret.jira_token.value
+    JIRA_USERNAME                       = var.jira_username
+    JIRA_CONTRACT_CUSTOM_FIELD          = var.jira_contract_custom_field
+    JIRA_DELEGATE_EMAIL_CUSTOM_FIELD    = var.jira_delegate_email_custom_field
+    JIRA_DELEGATE_NAME_CUSTOM_FIELD     = var.jira_delegate_name_custom_field
+    JIRA_ORGANIZATION_CF_CUSTOM_FIELD   = var.jira_organization_cf_custom_field
+    JIRA_ORGANIZATION_NAME_CUSTOM_FIELD = var.jira_organization_name_custom_field
+
+    # Apim connection
+    AZURE_APIM                = var.azure_apim
+    AZURE_APIM_RESOURCE_GROUP = var.azure_apim_resource_group
+    AZURE_SUBSCRIPTION_ID     = data.azurerm_subscription.current.subscription_id
+
+    AZURE_CLIENT_SECRET_CREDENTIAL_CLIENT_ID = data.azurerm_key_vault_secret.azure_client_secret_credential_client_id.value
+    AZURE_CLIENT_SECRET_CREDENTIAL_SECRET    = data.azurerm_key_vault_secret.azure_client_secret_credential_secret.value
+    AZURE_CLIENT_SECRET_CREDENTIAL_TENANT_ID = data.azurerm_client_config.current.tenant_id
+
+    # PostgreSQL 
+    REVIEWER_DB_HOST     = module.postgres_flexible_server_private.fqdn
```

Comment:
@balanza is `module.postgres_flexible_server_private.fqdn` the correct value?

Reply:
It should be good. In other configurations we manually formatted the hostname like in https://github.com/pagopa/io-infra/blob/4a8c6f3755e2b2805728516663a767d3518d8756/src/core/function_devportal_service_data.tf#L34